techniques/flash/FLASH25.html

```diff
@@ -105,7 +105,7 @@ <h3>Setting the accessible name through ActionScript 3.0</h3>
          
             <p>The code example below shows how a ListBox component is created and assigned an accessible name. </p>
          
-         <pre xml:space="preserve">import fl.controls.List;
+         <pre>import fl.controls.List;
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`